tests/scripts/components.sh

```diff
@@ -0,0 +1,4349 @@
+#! /usr/bin/env bash
```

Comment:
This file doesn't do anything useful when it's executed, so it should not have a shebang line and it should not be executable.